README.md

```diff
@@ -50,16 +54,32 @@ To enable logging in `DEBUG` level, add selected categories to list in `Help` ->
 The standard practice in Java logging is to use fully-qualified name of each class as category;
 in our case, however, we're only ever using the categories provided above.
 
+
 ### Generate plugin zip
 
 To generate a plugin archive run `:buildPlugin` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `buildPlugin` or `./gradlew buildPlugin`).
 The resulting file will be available under `build/distributions/`.
 
+
+### Run UI tests
+
+This repo ships with a JUnit driver fat jar (see `uiTests/lib/junit-driver-assembly-0.1.jar`) for UI tests,
+built using `sbt junitDriver/assembly` out of https://github.com/VirtuslabRnD/ide-probe/pull/97, commit `350189f7`,
+courtesy of [@lukaszwawrzyk](https://github.com/lukaszwawrzyk).
+
+UI tests are by default excluded from `./gradlew test` unless `runUiTests` property is set:
+
+```
+./gradlew -PrunUiTests test
```

Comment:
@mkondratek @micpiotrowski if you could run this stuff on your own machines

Reply:
see my comment below

Reply:
Comments addressed, let's see if @micpiotrowski will have any issues if he follows the updated guidelines

Reply:
`./gradlew -PrunUiTests test` ran without any issue

Reply:
but it was relatively fast (about one minute with compilation) so I'm not sure if everything went ok (was tested)

Reply:
This file is outdated... pls take a look into the updated README, it's `./gradlew -PenableUiTests --info test` now

Reply:
And it should open IntelliJ twice (once for 2019.3, and once for 2020.1) during the test run

Reply:
Yeah, my fault.
When I done this like in README everything goes fine